syntax = "proto3";

// Defining a service "Greeter" using proto file
service Greeter {
  // Unary RPC that receives HelloRequest and returns HelloReply
  rpc SayHello (HelloRequest) returns (HelloReply) {}
  // Unary RPC that receives HelloRequest and returns HelloReply
  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
}
// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
